### DISCUSSION TITLE
Challenge #1: Understanding Housing Inventory

### DISCUSSION THREAD

**Discussion by jacobrharris** [datakind/datakit-housing-fall-2024#1] [Challenges]
# Understanding Housing Inventory

In order to create policies and take action, housing actors need to understand the deficit–or surplus–in available housing units for households at different income levels. Without having this understanding, builders, policy-makers, and service providers do not have a clear understanding of how much inventory is needed, by when, to alleviate housing burdens in a community and support healthy community growth. In this challenge, you’ll work to generate insights, repeatable processes, and prototype tools for housing stakeholders to use to inform their community action. 

* The big question a housing stakeholder is trying to answer with the results of this challenge is: does my target community have a housing inventory surplus or deficit for my target service population?  
* The big question a community resident is trying to answer is: is there housing affordable to me in my target community?

## Community focus: Orlando

The [Christian Service Center](https://www.christianservicecenter.org), based in Central Florida, is looking to answer this specific inventory question for their service area of the MSA that is Orlando-Kissimmee-Sanford. [Eric Gray](https://www.orlandosentinel.com/2024/05/12/eric-gray-is-our-central-floridian-of-the-year/), Executive Director of the Christian Service Center has tried to answer this question in several ways and is looking to the DataKind community for help. Eric and his team will be expert resources throughout the DataKit. If you’re local to Orlando, The [Orlando Devs](https://www.eventbrite.com/o/orlando-devs-15266001174) will be arranging a campus tour during the DataKit \- stay tuned\!

## Get started with existing data

[Grab the data here](https://github.com/datakind/datakit-housing-fall-2024/releases/latest).

In order to understand housing inventory needs, housing actors must understand the population living in their community across income levels. Housing stakeholders utilize a spectrum from extremely low income to upper income as the population income distribution, as published by the US Dept of Housing and Urban Development (HUD.) This spectrum is anchored by HUD/US Census calculations for area median income (AMI). AMI is also called median family income (MFI). The five segments of the housing and income spectrum are as follows: 

1. Extremely low income: Below 30% of AMI  
2. Very low income: Below 50% of AMI  
3. Low income: Below 80% of AMI  
4. Moderate income: Between 80% and 120% of AMI  
5. Upper income: \>120% AMI

Published by Camoin Associates and cited by housing actors across communities, the [five-segment income and housing spectrum](https://camoinassociates.com/resources/humanizing-data-area-median-income-ami-and-affordable-housing-policy/) visualization is helpful for understanding target community populations across the spectrum. 

With existing data, you can start to identify the target service populations in a target community. To get started, you’ll create an understanding of population by segment in selected case study locations in the United States \- we’ve pulled data for Florida and California already and [added it here](https://github.com/datakind/datakit-housing-fall-2024/releases/latest). We encourage you to access more geographies through the [EODatascape](https://eodatascape.datakind.org). In your target geography, identify what the area median (family) income (AMI) is for that geography using the [HUD income limits tool](https://www.huduser.gov/portal/datasets/il/il2024/select_Geography.odn) ([check out](https://www.orangecountyfl.net/Portals/0/Library/Neighbors-Housing/docs/Income%20Limits%20Chart.pdf) how this information is used in City of Orlando planning). Geography is typically by state or by metropolitan statistical region (MSA) which is built from counties in a region. Once you have those limits, you’ll then need to calculate the income ranges for each segment of the income and housing population. 

Using the Median Household Income variables from the existing dataset, identify Census tract populations by housing segment. Consider evaluating this by Race (& Ethnicity), Gender and Age of Householder to inform insights around housing needs by population segments as well as income segments. 

Using the Median Monthly Housing Cost, assess the housing costs by Census tract and AMI. Consider evaluating based on homeownership. Summarize Census tract insights by county. 

This “getting started” analysis should help answer the following questions:

* What are the housing/income segments in the target community? What is the summary by county or state?  
* What is the geographic and demographic distribution of housing/income segments in the target community? Was it possible to identify Census tracts and populations within each of the five segments?  
* What is the homeownership and monthly housing cost by segment and geography in the target community?  
* Reflect: What surprised you from this analysis? What are some limitations of the analysis? What are ways to extend the work?

*Note \- [HUD’s LIHTC Qualified Census Tract](https://www.huduser.gov/portal/datasets/qct.html) program produces a related set of insights \- qualified Census tracts are identified as those containing “50 percent of households with incomes below 60 percent of the Area Median Gross Income (AMGI) or have a poverty rate of 25 percent or more.”* 

## Take it further

The “Getting started with existing data” analysis is helpful for understanding the target community, and to take it further, analysis will center on finding and assessing available housing inventory alongside the target community income and population information. 

Taking it further should help answer the following types of questions

* Research: how are other communities assessing inventory and responding to needs? What datasets are available to answer these questions?   
* Data analysis and visualization:   
  * What is the housing inventory in the target community? Conduct an assessment of the overall housing stock, including owner-occupied, rental, and vacant units.  
  * What is the housing inventory by housing/income segment?   
  * What are the housing inventory needs by housing/income segment?  
  * What is the availability of housing inventory (e.g. the unoccupied units)?  
  * What percentage of housing inventory is owned by private homeowners versus corporate or investor owners? Analyze the ratio of private (e.g., individual homeowners) to corporate-owned properties.  
  * How does the ownership structure affect housing availability? Investigate whether corporate-owned properties are more likely to be vacant or have higher rent, and whether they are concentrated in areas experiencing higher rates of displacement or cost burden.  
* Prototyping: What is the type of tool that might help a resident identify if there is available housing in their target community at the right income level?

## Additional data sets to explore

* [Housing | FRED | St. Louis Fed](https://fred.stlouisfed.org/categories/97)   
* [Housing Vacancies and Homeownership (CPS/HVS) \- Data](https://www.census.gov/housing/hvs/data/index.html)   
* [CoC Housing Inventory Count Reports \- HUD Exchange](https://www.hudexchange.info/programs/coc/coc-housing-inventory-count-reports/)  
* [Downloadable Housing Market Data \- Redfin](https://www.redfin.com/news/data-center/)  
* [Housing Data \- Zillow Research](https://www.zillow.com/research/data/)  
* [Low-Income Housing Tax Credit (LIHTC): Property Level Data | HUD USER](https://www.huduser.gov/portal/datasets/lihtc/property.html)  
* US Census (use 2022 5-year estimates where possible)  
  * [B25001 HOUSING UNITS](https://data.census.gov/table/ACSDT5Y2021.B25001?q=Housing+Units&g=040XX00US12$1400000)  
  * [H01 OCCUPANCY STATUS](https://data.census.gov/table/DECENNIALPL2020.H1?q=number%20of%20housing%20units%20by%20county)  
  * [B25008 TOTAL POPULATION IN OCCUPIED HOUSING UNITS BY TENURE](https://data.census.gov/table/ACSDT5Y2021.B25008?q=Housing+Units&g=040XX00US12$1400000)  
  * [B07013 GEOGRAPHICAL MOBILITY IN THE PAST YEAR BY TENURE FOR CURRENT RESIDENCE IN THE UNITED STATES](https://data.census.gov/table/ACSDT5Y2021.B07013?q=housing&g=040XX00US12$1400000)

**Comment by sana1410** [comment]
I have analyzed data for only target community which is **Orlando-Kissimmee-Sanford, FL MSA**  in florida contains the following areas: Lake County, FL; Orange County, FL; Osceola County, FL; and Seminole County, FL.

I have used the tool [HUD USER](https://www.huduser.gov/portal/datasets/il/il2022/2022summary.odn?year=2022&states=12.0&data=2022&inputname=METRO36740M36740*1209599999%2BOrange+County&stname=Florida&statefp=12&selection_type=county) tool for getting the income limits for the target community in year 2022.
![image](https://github.com/user-attachments/assets/599b25c0-bb3d-4ae7-be5e-9fcc5956d0b6)


I have used alteryx workflow to prepare data and create summary of following :-
- What are the housing/income segments in the target community? What is the summary by county or state?
- What is the geographic and demographic distribution of housing/income segments in the target community? Was it possible to identify Census tracts and populations within each of the five segments?(see attached excel file)
- What is the homeownership and monthly housing cost by segment and geography in the target community?

Answers :- 

What are the housing/income segments in the target community?
![image](https://github.com/user-attachments/assets/0aef3bdd-fa73-4b73-8ac2-4f0670f843db)

![image](https://github.com/user-attachments/assets/1bd5c93f-ba3b-420e-9a55-23e643852934)


What is the geographic and demographic distribution of housing/income segments in the target community?
![image](https://github.com/user-attachments/assets/3272d1e0-28f6-41ce-9cc2-bd31199fcb0c)

What are housing inventory and monthly housing cost for renter and owner for each segment in target community ?
![image](https://github.com/user-attachments/assets/b683b224-f3b9-43fa-a1e5-afae75f0e630)

![image](https://github.com/user-attachments/assets/d20a21d0-75b9-43d2-983f-72157acf5d8c)

![image](https://github.com/user-attachments/assets/684e4893-eab6-4b33-a933-67b66373bb2d)

Please find the files below 👎🏻 
[Challenge 1 output file.xlsx](https://github.com/user-attachments/files/17177611/Challenge.1.output.file.xlsx)
[output_file_orange_county.csv](https://github.com/user-attachments/files/17177612/output_file_orange_county.csv)
